src/lib/github/addIssuesToRepo.ts

```diff
@@ -9,20 +9,18 @@ export interface Issue {
 
  export default async function addIssuesToRepo(context: any, issues: Issue[] ): Promise<any> {
     const octokit = context.github;
-    return new Promise<any> ( () => {
-            const promises: Array<Promise<any>> = issues.map( async (issue: Issue) => {
-                const owner = context.payload.repository.owner.login;
-                const repo = context.payload.repository.name;
-                const title = issue.title;
-                const body = issue.body;
-                const fields = {
-                    owner,
-                    repo,
-                    title,
-                    body
-                };
-                return await octokit.issues.create(fields);
-            });
-            Promise.all(promises);
-    });
+        const promises: Array<Promise<any>> = issues.map( async (issue: Issue) => {
+            const owner = context.payload.repository.owner.login;
```

Comment:
The `owner` variable doesn't need to be in the map iteration loop. Please move it out of the map function and declare it out side it.